asteroidsyncserviced/main.cpp

```diff
@@ -41,5 +41,14 @@ Q_DECL_EXPORT int main(int argc, char *argv[])
     #endif
     DBusInterface *dbusInterface = new DBusInterface(watchesManager);
 
-    return a.exec();
+    int retval = a.exec();
+
+    if(dbusInterface)
+        delete dbusInterface;
```

Comment:
Actually there is no reason for these objects to be dynamically allocated. This would be cleaner:

```
WatchesManager watchesManager;
#ifdef UBUNTU_TOUCH_PLATFORM
UbuntuPlatform platform(watchesManager);
#elif SAILFISHOS_PLATFORM
SailfishPlatform platform(watchesManager);
#endif
DBusInterface dbusInterface(watchesManager);

return a.exec();
```

Then they would get trivially destroyed at the end of scope

Reply:
Done, (with address of operators).